zos_concepts/gdg_datasets/copy_edit_fetch/copy_edit_fetch.yml

```diff
@@ -0,0 +1,125 @@
+###############################################################################
+# © Copyright IBM Corporation 2024
+###############################################################################
+
+###############################################################################
+# This sample playbook demonstrates how to copy a directory to a PDS, edit a
+# member within the PDS and submit PDS member containing JCL.
```

Comment:
This description is used as a brief summary of the playbook, looks like it was not modified.